Monika After Story/game/00utils.rpy

```diff
@@ -309,3 +311,66 @@ python early in mas_utils:
     mas_log_open = mas_log.open()
     mas_log.raw_write = True
     mas_log.write("VERSION: {0}\n".format(renpy.game.persistent.version_number))
+
+    def deprecated(use_instead=None, should_raise=False):
+        """
+        Decorator that marks functions and classes as deprecated
```

Comment:
can you add a comment explaining that the deprecation warnings will appear in the mas_log when the code is **executed**, and not necessarily when its compiled to bytecode?